docs/api-guide.md

```diff
@@ -781,19 +785,29 @@ public class Sample {
 }
 ```
 
+The `begin()` API could throw `TransactionException` and `TransactionNotFoundException`.
+If you catch `TransactionException`, it indicates some failure (e.g., database failure and network error) happens during the transaction, so you should cancel the transaction or retry the transaction after the failure or error is fixed.
+If you catch `TransactionNotFoundException`, it indicates the transaction is not found, but it's retryable, so you can retry the transaction.
```

Comment:
> it indicates the transaction is not found

[minor] Is it possible to describe possible causes of the exception?

Reply:
Actually, this doesn't happens with ScalarDB, but It can happen with ScalarDB Cluster. So, I can't describe possible causes here... :(